argo-messaging.spec

```diff
@@ -0,0 +1,63 @@
+#debuginfo not supported with Go
+%global debug_package %{nil}
+
+Name: argo-messaging
+Summary: ARGO Messaging API for broker network
+Version: 0.9.0
+Release: 1%{?dist}
+License: ASL 2.0
+Buildroot: %{_tmppath}/%{name}-buildroot
+Group:     EGI/SA4
+Source0: %{name}-%{version}.tar.gz
+BuildRequires: golang
+BuildRequires: bzr
```

Comment:
I am not sure if we need bzr for this package. It seems we need `godep` or the package that provides `godep` if it's not part of `golang`